Dockerfile.armhf

```diff
@@ -0,0 +1,20 @@
+FROM resin/armhf-alpine:3.5
+
+ENV GOPATH /go
+ENV PATH $PATH:$GOPATH/bin
+
+RUN apk update -y
+RUN apk add --no-cache git ca-certificates go g++
```

Comment:
The reason do it this way is to build for every commit to the master, downloading only from the released binary won't produce right builds for building directly from master purposes.